### PR TITLE
docs: An example that implements broadcast communication

### PR DESCRIPTION
**Pull request by testwhygh**
It's a PR to solve the problem "[add an example to README as to how to broadcast a message to all worker threads using BroadcastChannel](https://github.com/piscinajs/piscina/issues/615)". This code has been tested,

### CODE REVIEW

**Review by metcoder95 — COMMENTED** [review]
This is amazing, thanks!
Could you rather add it to the `docs/examples` section?

**Review by ronag — COMMENTED** [review]
You should also make sure to set the flag that makes sure piscina doesn't use blocking Atomics.wait

**Comment by testwhygh** [comment]
> This is amazing, thanks! Could you rather add it to the `docs/examples` section?

Of course, I will also add it to `docs/docs/examples`

**Comment by testwhygh** [comment]
> You should also make sure to set the flag that makes sure piscina doesn't use blocking Atomics.wait

Yes, setting the flag `useAtomics` to false avoids using Atomics.wait, but why should I do this? Is it for better concurrency performance?

**Comment by metcoder95** [comment]
> > You should also make sure to set the flag that makes sure piscina doesn't use blocking Atomics.wait
> 
> Yes, setting the flag `useAtomics` to false avoids using Atomics.wait, but why should I do this? Is it for better concurrency performance?

Is to avoid the thread to be blocked when idle (we use `Atomic.wait` to put the thread on hold until further tasks are enqueued).

**Comment by testwhygh** [comment]
> > > You should also make sure to set the flag that makes sure piscina doesn't use blocking Atomics.wait
> > 
> > 
> > Yes, setting the flag `useAtomics` to false avoids using Atomics.wait, but why should I do this? Is it for better concurrency performance?
> 
> Is to avoid the thread to be blocked when idle (we use `Atomic.wait` to put the thread on hold until further tasks are enqueued).

Thanks, I have modified it.

**Review by metcoder95 — DISMISSED** [review]
lgtm

**Comment by testwhygh** [comment]
> lgtm
Hello, should I do anything else to get this PR completed?

File: README.md [hunk 1]
```diff
@@ -264,6 +264,56 @@ module.exports = ({ a, b }) => {
...
+```js
+'use strict';
+
+const { BroadcastChannel } = require('worker_threads');
+const { resolve } = require('path');
+
+const Piscina = require('piscina');
+const piscina = new Piscina({
+  filename: resolve(__dirname, 'worker.js'),
+  useAtomics: false
```
Comment (metcoder95) [inline README.md:281]: Let's just add a comment of why `useAtomics` is disabled, and it should be ready to go

Comment (testwhygh) [reply README.md:281]: Of course, it is necessary

**Review by metcoder95 — APPROVED** [review]

**Pull request merged**